MovieApp/MovieApp/Views/MoviePage/ViewModel/MoviePageViewModel.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+class MoviePageViewModel: MoviePageViewModelProtocol {
+    
+    // MARK: - Properties
+    
+    var onDataLoaded: (() -> Void)?
+    private var movieDetails: MovieDetails?
+    private let movieService: MovieServiceProtocol
+    private let dataService: RealmServiceProtocol
+    
+    init(movieService: MovieServiceProtocol, dataService: RealmServiceProtocol) {
+        self.movieService = movieService
+        self.dataService = dataService
+    }
+    
+    // MARK: - Public
+    
+    func getMovieDetails() -> MovieDetails? {
+        movieDetails
```

Comment:
2) Remove this method
3) Add movieDetails to the protocol `var movieDetails: MovieDetails? { get }`